app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentModel.kt

```diff
@@ -51,7 +54,7 @@ data class ScheduleAppointmentModel(
 
   val hasLoadedAppointmentFacility: Boolean
     get() = appointmentFacility != null
-
+  
```

Comment:
There seems to be an unnecessary space here, let's remove it

Reply:
There's no unwanted space. Please verify again

Reply:
Ah okay, looks like it's been resolved in later commits 👍🏽